Description: Fix minor spelling errors in readme and the manpage.
Subject: Fix minor spelling errors
Origin: vendor
Forwarded: no
From: Harlan Lieberman-Berg <H.LiebermanBerg@gmail.com>
Reviewed-by: gregor herrmann <gregoa@debian.org>
Last-Update: 2011-08-01

--- a/README
+++ b/README
@@ -88,7 +88,7 @@
     *   prefix - A string that will be automatically prepended to template
         names when searching for them in the path. Empty by default.
 
-    *   postfix - The default extension to be automtically appended to
+    *   postfix - The default extension to be automatically appended to
         template names when searching for them in the path. Don't forget to
         include the dot, such as '.html'. Empty by default.
 
@@ -200,11 +200,11 @@
     (which will simply be $filename with '.cache' appended to it.
 
   store_cachefile( $cachename, $template )
-    Receives the name of a template cache file and the corrasponding
+    Receives the name of a template cache file and the corresponding
     template object, and creates the cache file on disk.
 
   load_cachefile( $cachename, $template )
-    Receives the name of a template cache file and the corrasponding
+    Receives the name of a template cache file and the corresponding
     template object, reads the cache file and stores it in the template
     object (as 'script').
 
@@ -223,7 +223,7 @@
     and <http://www.kuwata-lab.com/tenjin/pltenjin-faq.html> for frequently
     asked questions.
 
-    Note that the Perl version of Tenjin is refered to as plTenjin on the
+    Note that the Perl version of Tenjin is referred to as plTenjin on the
     Tenjin website, and that, as opposed to this module, the website
     suggests using a .plhtml extension for the templates instead of .html
     (this is entirely your choice).
@@ -251,7 +251,7 @@
         present.
 
     Version 0.06 (this version) restored the layout template feature which
-    was accidentaly missing in version 0.05, and the ability to call the
+    was accidentally missing in version 0.05, and the ability to call the
     utility methods of Tenjin::Util natively inside templates. You will want
     to remove your templates' .cache files when upgrading to 0.6 too.
 
@@ -263,7 +263,7 @@
     Development of Tenjin is done with github at
     <http://github.com/ido50/Tenjin>.
 
-ACKNOWLEDGEMENTS
+ACKNOWLEDGMENTS
     I would like to thank the following people for their contributions:
 
     *   Makoto Kuwata
--- a/lib/Tenjin.pm
+++ b/lib/Tenjin.pm
@@ -122,7 +122,7 @@
 =item * B<prefix> - A string that will be automatically prepended to template names
 when searching for them in the path. Empty by default.
 
-=item * B<postfix> - The default extension to be automtically appended to template names
+=item * B<postfix> - The default extension to be automatically appended to template names
 when searching for them in the path. Don't forget to include the
 dot, such as '.html'. Empty by default.
 
@@ -499,7 +499,7 @@
 =back
 
 Version 0.06 (this version) restored the layout template feature which was
-accidentaly missing in version 0.05, and the ability to call the utility
+accidentally missing in version 0.05, and the ability to call the utility
 methods of L<Tenjin::Util> natively inside templates. You will want to
 remove your templates' .cache files when upgrading to 0.6 too.
 
@@ -593,4 +593,4 @@
 
 See http://dev.perl.org/licenses/ for more information.
 
-=cut
\ No newline at end of file
+=cut
